exploration/beauty-contest.md

```diff
@@ -0,0 +1,316 @@
+# Syntax Variation Beauty Contest
+
+As discussed in the 2023-09-25 teleconference, we need to choose a syntax to proceed with.
+This page hosts various options for considering in the 2023-10-02 call.
+
+## 0. Current
+
+This is the current syntax,
+including the changes introduced in [#488](https://github.com/unicode-org/message-format-wg/pull/488).
+The list of messages in this section also serves as the basis for all other examples.
+
+```
+{Hello world!}
+
+{Hello {$user}}
+
+input {$var :function option=value}
+{Hello {$var}}
+
+input {$var :function option=value}
+local $foo = {$bar :function option=value}
+{Hello {$var}, you have a {$foo}}
+
+match {$foo} {$bar}
+when foo bar {Hello {$foo} you have a {$var}}
+when * * {{$foo} hello you have a {$var}}
+
+match {$foo :function option=value} {$bar :function option=value}
+when a b {  {$foo} is {$bar}  }
+when x y {  {$foo} is {$bar}  }
+when * * {  {$foo} is {$bar}  }
+
+input {$var :function option=value} local $foo = {$bar :function option=value}{Hello {$var}, you have a {$foo}}
+
+match {$foo} {$bar} when foo bar {Hello {$foo} you have a {$var}} when * * {{$foo} hello you have a {$var}}
+
+match {$foo :function option=value} {$bar :function option=value}when a b {  {$foo} is {$bar}  }when x y {  {$foo} is {$bar}  }when * * {  {$foo} is {$bar}  }
+```
+
+## 1. Invert for Text Mode
+
+Consumes exterior whitespace.
+
```

Comment:
I don't think this is a correct statement. In fact, you can see unquoted literal strings in all of the `when` clause examples. Can you explain what you mean?

Reply:
Ah, good point. I fixed the suggestion above to specify unquoted literal _operands_, which we can't have because they would conflict with e.g. `{input` and `{when`.